src/librustdoc/html/render/mod.rs

```diff
@@ -1115,7 +1119,9 @@ fn render_assoc_items_inner(
         }
         if !impls_buf.is_empty() {
             w.push_buffer(tmp_buf);
+            write!(w, "<div id=\"{}-list\">", id);
```

Comment:
I see that you manually added `implementations-list` to the id_map, and the ID of the deref H1 items gets added to the id_map by `derive_id`. But I can't find anywhere that the modified ID. where you take a deref version and add `-list` to the end, that gets added to the id_map.

It seems like I could engineer an ID conflict by adding `# Deref Methods U8 List` as a header to my markdown.

Reply:
Good point! I'll add a `derive_id` for it.